lib/lrama/lexer.rb

```diff
@@ -257,6 +288,9 @@ def lex_user_code(ss, line, column, lines)
         when ss.scan(/\$(<[a-zA-Z0-9_]+>)?(\d+)/) # $1, $2, $<long>1
           tag = ss[1] ? create_token(Token::Tag, ss[1], line, str.length) : nil
           references << [:dollar, Integer(ss[2]), tag, str.length, str.length + ss[0].length - 1]
+        when ss.scan(/\$(<[a-zA-Z0-9_]+>)?([a-zA-Z0-9_]+)/) # $foo, $expr, $<long>program
```

Comment:
Sorry, exactly saying ...
It might be same with one of two:
1. `Token::Ident` (`[a-zA-Z_.][-a-zA-Z0-9_.]*`)
2. `Token::Named_Ref` except for `[]` (so that `[a-zA-Z_.][-a-zA-Z0-9_.]*`)

Reply:
Thank you for reviewing. Is this the change you were suggesting?
https://github.com/ruby/lrama/pull/41/commits/3b8d4e585653ddf07273f4ee9a303d69a9daa38a#diff-816037370d9ebaadd970a429ac66cdf5597623e3f285642396489c85b05b89c5R266

Reply:
Yes, do you have any concern?